app/src/main/AndroidManifest.xml

```diff
@@ -2,13 +2,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.teamtwo.apilol">
 
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.INTERNET"/>
+
     <application
+        android:usesCleartextTraffic="true"
```

Comment:
¿Es necesaria esta línea? El indicar esto en el manifest normalmente indica que la aplicación se está conectando a servidores inseguros y a Google no suele gustarle...

Reply:
Esto fue porque de primeras me daba un error al conectar con la api, después ya fue cuando me dió el error del parseo pero lo quito y pruebo

Reply:
Si no se lo dejo activo me ocurre lo siguiente:

CLEARTEXT communication to ddragon.leagueoflegends.com not permitted by network security policy

Reply:
En tu llamada estas usando http o https? me suena que ese error es por intentar llamar a una url http y a partir de Android 8 no se puede por defecto.

Reply:
Correcto, solucionado. He añadido el https y se acabó el problema. GRACIAS!!